
	{% extends './layout/sub_layout.html' %}

	{% block head%}
		<script type="text/javascript">

		</script>
	{% endblock %}


	<!-- 컨텐츠 컨테이너 시작-->
	{% block ContentContainer %}
	<div class="container pfarticipation">
		<div class="subVisual inquiry">
			<div class="mv_content">
				<div class="text_area">
					<span>Tutorial</span>
				</div>
			</div>
		</div>
		
		<!-- 제목 시작 -->
		{% block titleArea %}
			<div class="title_area">
				<span>AIVE Tutorial</span>
			</div>
		{% endblock %}
		<!-- //제목 종료 -->
		
		
		<!-- topmenu start -->
		{% block mainContent%}
		<div class="board_area" >
			<!--슬라이드2 시작-->
			<div class="tutorial_img01">
				<img src="/static/img/tutorial/img_tutorial_top.png" />
			</div>
				<p span class="title_num2">(1) AIVE can be used to evaluate :</p>
			<div class="tutorial_expl02">
				<dt>
					A. Protein structure prediction from viral amino acid sequences<p></p>
					B. Analysis of repeated polarity in viral amino acid sequences (PS)<p></p>
					C. Rate of change of mutations for viral genome sequences (MR)<p></p>
					D. Scoring of amino acid changes based on amino acid properties (BPES)<p></p>
					E. Intergrative mathematical analysis of protein structure prediction (amino acid clustering), PS, MR, and, BPES
				</dt> 
			</div>
			<p span class="title_num2">(2) The input range of analysis for AIVE is as follows :
			<div class="tutorial_expl02">
				<dt>
					A. Basically provided SARS-CoV-2 VOCs
				</dt> 
				<dl>
					<dd>
						- Monomer (single amino acid sequence)<p></p>
						- Multimer (two or more amino acid sequences, when analyzing docking score, only two sequences are possible)
					</dd>
				</dl>
				<dt>
					<p></p>
					B. Analysis of amino acid sequences written by the user
				</dt> 
				<dl>
					<dd>
						- Monomer (single amino acid sequence)<p></p>
						- Multimer (two or more amino acid sequences, when analyzing docking score, only two sequences are possible)<p></p>
						- PDB and JSON files that users possess can be directly submitted
					</dd>
				</dl>
		    </div>
			<div class="tutorial_step">
				Step 1. Sign up & Login
			</div>
			<p class="title_when2">Sign up</p>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						If the user does not have an account, click on Sign Up  to create one then proceed to log in.
					</dd>
				</dl>
		   	</div>
		   	<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_signup.png" />
				</div>
			</div>
			<p class="title_when2">Login</p>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						Click on the Login button in the upper right corner of the AIVE page to log in.
					</dd>
				</dl>
		   	</div>
		   	<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_login.png" />
				</div>
			</div>

			<div class="tutorial_step">
				Step 2. Input
			</div>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						- Click on Prediction from the menu.<p></p>
						- Click on Local from the submenu to predict project by using AlphaFold2.<p></p>
						- Click on Colab from the submenu to predict project by using AlphaFold Colab.
					</dd>
				</dl>
		   	</div>
		   	<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Predictionlist.png" />
				</div>
			</div>
			<p class="title_when2">RUNNING THE DEMO</p>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						The AIVE demo has been prepared for the user’s convenience. For the RBM section of omicron BA.5, the representative SARS-CoV-2 strain, input the amino acid sequence, run, and output the result. The demo result can be viewed in job list.
					</dd>
				</dl>
			</div>
			<p></p>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						Click on the DEMO button
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Prediction_demo_btn.png" />
				</div>
			</div>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						Click on the Server Prediction button
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Prediction_server_btn.png" />
				</div>
			</div>

			<p class="title_when2">RUNNING SARS-CoV-2 VOC</p>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						- Input the project name to categorize the submitted works.<p></p>
						- From Target virus, select SARS-CoV-2 and VOC<p></p>
						- Click the domain that the user wants to predict<p></p>
						- Click on the Server Prediction button
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Prediction_virus1.png" />
				</div>
			</div>

			<p class="title_when2">RUNNING YOUR SEQUENCE OR FASTA FILE</p>
			<div class="tutorial_expl02">
				<dt>
					Input virus sequence
				</dt> 
				<dl>
					<dd>
						- Input the project name about to be analyzed into Project name.<p></p>
							: Write the project name to categorize the submitted works.<p></p>
						- From Target virus, select the desired virus<p></p>
							: From Target virus, select the virus that the user wants to check. If the virus is not on the list, select All Viruses.<p></p>
							- Enter the virus amino acid sequence that the user wants to predict into the Input virus Sequence column.<p></p>
							- To carry out Complex model predictions (multiple prediction), click on +Add on the right to add a space to enter sequences. Click on the –Del button to remove it.<p></p>
							- Click on the Upload fasta file button to submit fasta files to enter Sequences.<p></p>
					</dd>
				</dl>
			</div>
			<div class="tutorial_expl02">
				<dt>
					Generate mutation
				</dt> 
				<dl>
					<dd>
						The sequence entered into the Input virus sequence tab gets output in Alignment.<p></p>
						- To create mutations in Protein sequence, click on the corresponding location to select one from 20 amino acids. Select one of the codons that comprise the selected amino acid<p></p>
						- In Variants protein sequence, sequences with mutations applied are output.<p></p>
						- Click on the Server Prediction button to start generated  sequence structure prediction in the server and move to the job list page.<p></p>
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Prediction_virus2.png" />
				</div>
			</div>
			<p class="title_when2">RUNNING Prediction in AlphaFold Colab</p>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						Step 1. When using Colab, please adjust settings in the following order<p></p>
						Step 1-a. Navigate to colab / edit / notebook settings, edit hardware accelerator settings<p></p>
						Step 1-b. For the hardware accelerator settings, select GPU. Colab pro users can select high-RAM under Runtime Shape settings<p></p><p></p>
						Step 2. Proceed to AlphaFold Colab, and carry out the following steps<p></p>
						Step 2-a. Before running 3. Enter the amino acid sequence(s) to fold, insert the sequence first<p></p>
						Step 2-b. Run in the following order: 1. install third-party software -> 2. Download AlphaFold -> 3. Enter the amino acid sequence(s) to fold -> 4. Search against genetic databases -> 5. Run AlphaFold and download prediction<p></p><p></p>
						Step 2-c. Predicition.zip file is saved<p></p>
						Step 2-d. After extracting the Predicition.zip file, .PDB and .JSON files are created
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Prediction_alphafold.png" />
				</div>
			</div>

			<div class="tutorial_step">
				Step 3. Result page
			</div>
			<p class="title_when2">Navigate to job list page</p>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						- Click on Report from the menu.<p></p>
						- Click on List from the submenu to navigate to Job List.<p></p>
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Report_joblist1.png" />
				</div>
			</div>
			<p class="title_when2">Navigate to job list page</p>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						- This is the submitted Job List. The columns displayed include the project name of the submitted process, whether it’s a monomer prediction or multimer prediction, which virus was selected for target virus, process status, and result page link.<p></p>
						- When the structure prediction is complete, click on the Result link to go to the report page.
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Report_joblist2.png" />
				</div>
			</div>
			<p class="title_when2">Result report page – Predicted protein structure</p>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						① In the case of Local prediction, 5 prediction models are created. When a number between 1 to 5 is applied, it can be checked based on the average pLDDT value.<p></p>
						② 3D structure model of the protein model selected in ① is visualized.<p></p>
						③ Predicted aligned error (PAE)<p></p>
						 - PAE is a value that estimates the difference between the relative locations of two residues of the model and the real model. A low PAE value indicates that the accuracy of the relative location of the two residues is high.<p></p>
						 • The color at (x, y) indicates AlphaFold’s expected position error at residue x if the predicted and true structures were aligned on residue y.<p></p>
						 • If the PAE is generally low for residue pairs x, y from two different domains, it indicates that AlphaFold predicts well-defined relative positions and orientations for them.<p></p>
						 • If the PAE is generally high for residue pairs x, y from two different domains, then the relative positions and/or orientations of these domains in the 3D structure are uncertain and should not be interpreted. (Explanation from AlphaFold FAQ)<p></p>
						④ Predicted LDDT<p></p>
						 - pLDDT is a value that estimates the reliability of the model. It estimates how well the actual model residue and predicted model residue match. At the same time, it indicates how well the protein structure folds in the corresponding location. A low pLDDT value indicates that the reliability of the corresponding position is low and that it possesses a disordered structure<p></p>
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Report_3D.png" />
				</div>
			</div>
			<p class="title_when2">Result report page – evaluate model</p>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						For the amino acid sequence entered by the user, the AIVE system provides a total of 6 evaluation charts. First, AIVE predicts protein structures due to mutations in each gene for coronavirus lineages or sub-lineages. From the predicted result, it carries out grouping of amino acids (components of 3D protein structure) utilizing K-means clustering to report SCPS results. Second, through repeated pattern analysis of polar amino acids in amino acid sequences, AIVE reports PS results. Third, AIVE figures out amino acid properties to measure BPES through measurement of changes in biochemical properties of amino acid. Fourth, AIVE calculates MR through rate of change for nucleotide frequencies due to mutations. Fifth, APESS, the result of comprehensive mathematical model (SCP*PS*MR*BPES) of measured analysis results is provided. Finally, the APESS distribution graph provides risk and spread results of the amino acid sequence entered by the user by comparing to VOCs’ APESS evaluation metric.
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Reprot_evaluate_model.png" height="800"/>
				</div>
			</div>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						SCPS(SubClustering of Protein Structure)<p></p>
						: SCPS divides the residues constituting the 3D protein structure into several groups using K-means clustering with protein structure prediction information(PAE, pLDDT). The indicated groups are those with a high proportion of WHO name variants.
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Reprot_evaluate_model1.png" />
				</div>
			</div>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						PS(Polarity Score)<p></p>
						: PS assigns a weight to a position having a specific structure (where P appears consecutively) through the polarity of amino acid structure information constituting the protein structure.
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Reprot_evaluate_model2.png" />
				</div>
			</div>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						BPES(Biochemical Properties Eigen Score)<p></p>
						: BPES measures changes in the biochemical properties of amino acids at the site of a mutation. A high rate of change results in a high BPES value. Biochemical properties are measured by integrating amino acid residue, pH, and hydrophobic information.
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Reprot_evaluate_model3.png" />
				</div>
			</div>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						MR(Mutation Rate)<p></p>
						: MR is the rate of change of Amino Acid and its constituent nucleotides; the higher the rate, the higher the rate of change.
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Reprot_evaluate_model4.png" />
				</div>
			</div>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						APES<p></p>
						: APESS is a value that integrates the results of SCPC, PS, BPES, and MR, and the higher the value, the more dangerous the mutation is.
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Reprot_evaluate_model5.png" />
				</div>
			</div>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						APESS distribution graphp</p>
						: The graph shows the distribution of APESS scores of variants generated through random sampling, and the dotted line indicates the quantile set to 0.05. The graph indicates the score positions of the WHO VOC variants, and a thick line indicates the APESS score position of the predicted protein
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Reprot_evaluate_model6.png" />
				</div>
			</div>
			
			<p class="title_when2">Result report page – polarity</p>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						Amino acid polarity affects protein structure and stability. As a result, the amino acid polarity due to mutation of the amino acid sequence input by the user can be observed. We found repeated polarity patterns in the coronavirus and observed changes in the properties of amino acid sequence polarity due to mutation. Therefore, we provide visualization and table view of polarity pattern changes to the user.
					</dd>
				</dl>
			</div>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						- First line: amino acid sequence<p></p>
						- Second line: 4 polarity characteristics<p></p>
						- Third line: 5 amino acid properties<p></p>
						- Mutated positions for wild type virus sequencing are indicated in red.<p></p>
						- Mutated positions for mutated type virus sequencing are indicated in red.<p></p>
						- Polarity and amino acid properties classification table
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Report_polarity.png" />
				</div>
			</div>
			<p class="title_when2">Submit your result file</p>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						Users can input PDB, PAE JSON files into the AIVE system to return the final result, APESS score.
					</dd>
				</dl>
			</div>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						- Click on Report from the menu.<p></p>
						- Click on Result viewer from the submenu.<p></p>
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Report_viewer.png" />
				</div>
			</div>
			<div class="tutorial_expl02">
				<dl>
					<dd>
						- If you want compare with SARS-CoV-2 VOC result, select SARS-CoV-2 and VOC<p></p>
						- Upload your PDB result file<p></p>
						- Upload your PAE JSON file<p></p>
						- Click on Show result<p></p>
					</dd>
				</dl>
			</div>
			<div class="tutorial_img2are">
				<div class="tutorial_img01">
					<img src="/static/img/tutorial/tutorial_Report_result.png" />
				</div>
			</div>
		{% endblock %}
	</div>
	{% endblock %}

	


